apps/accounts/tests/test_login.py

```diff
@@ -65,34 +64,3 @@ def test_logout(self):
         response = self.client.get(reverse('mylogout'), follow=True)
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'Login')
-
-    def test_settings_auth_results_in_usertype_benny(self):
```

Comment:
In case there's some confusion here, it turns out this test isn't doing anything. The if statement returns False in all our testing environments, so it goes to the else statement, and then checks True==True. The response could 404 and this would still pass.